src/models/network.ts

```diff
@@ -1,5 +1,5 @@
 export enum Network {
-  MainnetProd,
-  DevnetProd,
-  DevnetStaging,
+  Mainnet,
+  Devnet,
+  Localnet,
```

Comment:
Do we need localnet here when we don't support it yet? We can add it in when we support it

Reply:
Wdym we don't support it? We can support any network.

I.e we should keep this so that we can write integration tests against a local program. (A barebones version of our program tests)

Reply:
No I meant we don't have configs